.github/workflows/conda-package.yml

```
@@ -257,7 +257,8 @@ jobs:
       - name: Run tests
         if: env.RERUN_TESTS_ON_FAILURE != 'true'
         run: |
-          python -m pytest -q -ra --disable-warnings -vv ${{ env.TEST_SCOPE }}
+          export DPNP_TEST_ALL_TYPES=1
+          python -m pytest -n auto -q -ra --disable-warnings -vv ${{ env.TEST_SCOPE }}
         working-directory: ${{ env.tests-path }}
```

The same can be done through `env` option:
```suggestion
          python -m pytest -n auto -q -ra --disable-warnings -vv ${{ env.TEST_SCOPE }}
        working-directory: ${{ env.tests-path }}
        env:
          DPNP_TEST_ALL_TYPES: 1
```